internal/resources/resource_host.go

```diff
@@ -579,12 +559,28 @@ func resourceMetalHostUpdate(d *schema.ResourceData, meta interface{}) (err erro
 		}
 	}
 
+	// description
+	updDesc, ok := d.Get(hDescription).(string)
+	if ok && updDesc != host.Description {
+		host.Description = updDesc
+	}
+
 	// initiator name
 	updInitiatorName, ok := d.Get(hInitiatorName).(string)
 	if ok && updInitiatorName != "" && updInitiatorName != host.ISCSIConfig.InitiatorName {
 		host.ISCSIConfig.InitiatorName = updInitiatorName
 	}
 
+	// set the network ids
+	if host.NetworkIDs, err = getNetworkIDs(d, p, &host); err != nil {
+		return err
+	}
```

Comment:
while more code, may be better to do 
ids, err := getNetworkIDs(...)
if err != nil {
   return err
}

host.NetworkIDs = ids

Reply:
Why the need for 'ids'? If an error occurs, does it matter what 'host.NetworkIDs' is set to? Could you provide context on the 'better' part?

Reply:
perhaps not an issue in this case, but the assignment even when there's an error is a pattern that can be less than obvious